src/main/java/com/purbon/kafka/topology/model/users/Consumer.java

```diff
@@ -1,13 +1,18 @@
 package com.purbon.kafka.topology.model.users;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.purbon.kafka.topology.model.User;
+import com.purbon.kafka.topology.roles.rbac.RBACPredefinedRoles;
 import java.util.Objects;
 import java.util.Optional;
 
 public class Consumer extends User {
 
   private Optional<String> group;
 
+  @JsonProperty(value = "group-role")
+  private String groupRole = RBACPredefinedRoles.RESOURCE_OWNER;
```

Comment:
This model is used for both RBAC and ACLs permissions and might be even used later on for other integrations (I know some people who tried to do it with Ranger, for example).

I think we should not be using here:

* The variable name as _group-role_, as with ACLs there are no roles per se.
* The default value as RESOURCE_OWNER as it is Confluent RBAC dependant.

What do you think?

Reply:
I agree, I will try to see how to deal with this point.

Reply:
If we add a level called "rbac" in the configuration of a consumer, would it be okay for you ?

```yaml
consumers:
     - principal: "Group:CONSUMER_READ"
       rbac:
           role: DeveloperRead
     - principal: "Group:CONSUMER_VIEW"
       rbac:
            role: DeveloperRead
```


We should do the same in the schema configuration I guess, even if currently, ACLs are only available with RBAC for schemas. 
WDYT ?

Reply:
@purbon kind reminder on the proposal :)